components/permit-holders/modals/EditUserInformationModal.tsx

```diff
@@ -115,11 +130,7 @@ export default function EditUserInformationModal({
                 <Stack direction="row" spacing="20px">
                   <FormControl isRequired>
                     <FormLabel>{`Date of birth`}</FormLabel>
-                    <Input
-                      type="date"
-                      value={dateOfBirth}
-                      onChange={event => setDateOfBirth(event.target.value)}
-                    />
+                    <DatePicker width="100%" value={dateOfBirth} onDateChange={setDateOfBirth} />
```

Comment:
I'm not super sure how I feel about using this DatePicker component here. The component doesn't allow for changing the year, only the month, and since birthdays will probably be a long time ago it would take a lot of scrolling back through the months to get to the right year. Because of this, users would probably just end up fully typing the date anyways. Is there anyway we could add editing just the year in the date picker instead of only the month?

Reply:
That's a really good point. I just tested it as well, and it's not possible to enter the date manually as the component doesn't take in input for some reason. The old implementation was working fine and easily allowed manually entering the date, so I'll revert to the old version for now. We will probably style it manually in the near future to follow our design system.

We could also add in year editing through this example https://react-day-picker.js.org/examples/elements-year-navigation/ but that would require a new design as well. Also, from a UX standpoint, I don't remember the last time I selected my DOB on a website through a date picker - I just used the date input so I feel like manually entering the date would be the main input method here.

Will bring this up with product team too to get their thoughts.

Reply:
Reverting back to the HTML date picker input for now, which we will likely stick with. Any changes will be addressed in a future PR.